pages/windows/expand.md

```diff
@@ -0,0 +1,23 @@
+# expand
+
+> Uncompress one or more Windows Cabinet files.
+
+- Uncompress a single-file Cabinet file to the specified directory:
+
+`expand {{path/to/file.cab}} {{path/to/directory}}`
+
+- Display a list of files in the source Cabinet file:
```

Comment:
Display _the_ list of files in _a_ source Cabinet file ?

Reply:
Yeah, that sounds better. 👍